code/modules/nano/nanoui.dm

```diff
@@ -253,8 +263,9 @@ nanoui is used to open and update nano browser uis
   *
   * @return nothing
   */
-/datum/nanoui/proc/add_script(file)
-	scripts.Add(file)
+/datum/nanoui/proc/add_script(name, file)
+	scripts["[ckey(name)].js"] = file
```

Comment:
This seems weird, could you please explain why you’ve made this change?
Does every client need their own version of a js file?